plugins/woocommerce-admin/client/dashboard/components/connect/index.js

```diff
@@ -7,7 +7,7 @@ import { Component, Fragment } from '@wordpress/element';
 import { compose } from '@wordpress/compose';
 import PropTypes from 'prop-types';
 import { withDispatch, withSelect } from '@wordpress/data';
-import { PLUGINS_STORE_NAME } from '@woocommerce/data';
+import { ONBOARDING_STORE_NAME } from '@woocommerce/data';
```

Comment:
I am not sure if it is a good idea to use the "onboarding" data store. It works great this way but I am not sure if by using something that is meant for "onboarding" (so in my understanding: the fresh install setup wizard) we won't be disruptive?

Reply:
This is ok -- we consider the task list to be part of onboarding.